src/middleware.js

```diff
@@ -43,17 +43,22 @@ export default function ({ onUnauthorized, ...options }) {
       headers,
       method,
       mode,
+      actions,
       types,
       url,
       ...rest,
     } = lpApi
 
-    const [ requestType, successType, errorType ] = types
+    // Alias 'actions' with 'types' for backwards compatibility
+    const [ requestAction, successAction, errorAction ] = actions || types
```

Comment:
good point. i suppose you could 'parse' them into a generic action creator that takes the payload and error status, then use that going forward. Not sure if that makes things simpler or more complicated. Good w me either way.